packages/cmr-client/CMRSearchConceptQueue.js

```diff
@@ -33,13 +33,15 @@ class CMRSearchConceptQueue {
    * @param {string} params.format - the result format
    */
   constructor(params = { searchParams: {} }) {
-    this.clientId = params.clientId;
-    this.provider = params.provider;
+    this.clientId = params.cmrSettings.clientId;
+    this.provider = params.cmrSettings.clientId;
```

Comment:
Shouldn't this be `params.cmrSettings.provider`? Is `this.provider` even used anywhere? Seems like 40 just refers directly to `params.cmrSettings.provider`?

Reply:
Thank you. fixed.